autorest/preparer_test.go

```diff
@@ -70,7 +70,7 @@ func ExampleCreatePreparer() {
 	} else {
 		fmt.Println(r.URL)
 	}
-	// Output: https://microsoft.com/a/b/c
```

Comment:
@brendandixon We are using url.Opaque now and url.String() uses below form if url.Opaque is non-empty.

```scheme:opaque?query#fragment       ---- no host ```